pkg/tests/docker-compose-test.yaml

```
@@ -0,0 +1,16 @@
+db:
+    container_name: totem-pedidos-db
+    image: postgres:16-alpine
+    ports:
+      - "5432:5432"
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)